src/lib/fauna-import-writer.js

```diff
@@ -25,6 +26,7 @@ function getFaunaImportWriter(
   inputFile,
   isDryRun = false,
   logger = console.log,
+  allowRetries = false,
```

Comment:
is the reason this is a parameter to keep existing tests functional?

IMO we'd be better off just always retrying and updating our tests to account for this.